scripts/jsr-sync-versions.ts

```diff
@@ -8,41 +8,137 @@ const __dirname = path.dirname(__filename)
 const rootDir = path.resolve(__dirname, '..')
 const packagesDir = path.join(rootDir, 'packages')
 
-async function syncJsrVersions() {
+const SUSPENSIVE_SCOPE = '@suspensive'
+
+function convertToJsrCompatibleVersion(version: string): string {
+  if (version.includes('||')) {
+    const versions = version.split('||')
+    return versions[versions.length - 1].trim()
+  }
+
+  if (version === '*') {
+    return 'latest'
+  }
+
+  return version
```

Comment:
This is the part that is causing the problem. Expressing dependencies in jsr is harder than I thought.